src/main/java/com/laa/model/crime/CaseType.java

```diff
@@ -0,0 +1,5 @@
+package com.laa.model.crime;
+
+public enum CaseType {
+    INDICTABLE,  TRIAL_IN_CROWN_COURT, SUMMARY_ONLY
```

Comment:
I thought cases can only be Indictable, summary and either way.  Indictable and trial in crown court are the same thing

Reply:
INDICTABLE is meg's court, these given by business.